### ISSUE TITLE
@babel/types v7.13.12 having install issues

### ISSUE THREAD

**Issue by aarondack** [babel/babel#13039]
## Bug Report

Hello hello, 

Just wanted to let you all know `@babel/types@7.13.12` is having issues. Looks like it also has zero downloads on npm as per below which seems quite suspect.

![image](https://user-images.githubusercontent.com/8890348/112054394-82d44e00-8b23-11eb-8b0c-bb4c65815f59.png)

**Current behavior**

Installing seems to not match a version.

![image](https://user-images.githubusercontent.com/8890348/112054492-aa2b1b00-8b23-11eb-8f38-faf8cb9bb886.png)

**Environment**

```

```
- Babel version(s): [e.g. v7.12.0]
- Node/npm version: Node 14
- OS: macOS 10.15.7
- Monorepo: no
- How you are using Babel: webpack

**Possible Solution**

**Additional context**

**Comment by JLHwung** [comment]
This is correctly published on https://www.npmjs.com/package/@babel/types

It's likely a problem with a cache between your CI build and the npm registry.

See also https://github.com/babel/babel/issues?q=label%3A%22i%3A+npm+cache%22+is%3Aclosed

**Comment by alesniak303** [comment]
Same problem here.  It appears that @babel/types was published after @babel/helper-module-imports, and that may be causing issue.

**Comment by nicolo-ribaudo** [comment]
@alesniak303 `@babel/types` was published before :thinking: 

They were published in this order:
```
  ➤ YN0000: │ Published @babel/compat-data 7.13.12
  ➤ YN0000: │ Published @babel/helper-transform-fixture-test-runner 7.13.12
  ➤ YN0000: │ Published @babel/node 7.13.12
  ➤ YN0000: │ Published @babel/parser 7.13.12
  ➤ YN0000: │ Published @babel/plugin-proposal-optional-chaining 7.13.12
  ➤ YN0000: │ Published @babel/plugin-transform-proto-to-assign 7.13.12
  ➤ YN0000: │ Published @babel/types 7.13.12
  ➤ YN0000: │ Published @babel/standalone 7.13.12
  ➤ YN0000: │ Published @babel/helper-member-expression-to-functions 7.13.12
  ➤ YN0000: │ Published @babel/helper-module-imports 7.13.12
  ➤ YN0000: │ Published @babel/helper-simple-access 7.13.12
  ➤ YN0000: │ Published @babel/helper-define-map 7.13.12
  ➤ YN0000: │ Published @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining 7.13.12
  ➤ YN0000: │ Published @babel/preset-env 7.13.12
  ➤ YN0000: │ Published @babel/helper-replace-supers 7.13.12
  ➤ YN0000: │ Published @babel/plugin-transform-react-jsx 7.13.12
  ➤ YN0000: │ Published @babel/helper-module-transforms 7.13.12
```

Source: https://github.com/babel/babel/runs/2167476229?check_suite_focus=true (expand the "Publish to npm" step).

**Comment by nicolo-ribaudo** [comment]
Anyway, these are problems caused either by proxies between the npm registry and your machines, or by cache bugs in the npm registry itself (they have been happening for years, but it's really hard for them to debug and fix them).

There is nothing actionable from us here :shrug:

**Comment by alesniak303** [comment]
> @alesniak303 `@babel/types` was published before 🤔
> 
> They were published in this order:
> 
> ```
>   ➤ YN0000: │ Published @babel/compat-data 7.13.12
>   ➤ YN0000: │ Published @babel/helper-transform-fixture-test-runner 7.13.12
>   ➤ YN0000: │ Published @babel/node 7.13.12
>   ➤ YN0000: │ Published @babel/parser 7.13.12
>   ➤ YN0000: │ Published @babel/plugin-proposal-optional-chaining 7.13.12
>   ➤ YN0000: │ Published @babel/plugin-transform-proto-to-assign 7.13.12
>   ➤ YN0000: │ Published @babel/types 7.13.12
>   ➤ YN0000: │ Published @babel/standalone 7.13.12
>   ➤ YN0000: │ Published @babel/helper-member-expression-to-functions 7.13.12
>   ➤ YN0000: │ Published @babel/helper-module-imports 7.13.12
>   ➤ YN0000: │ Published @babel/helper-simple-access 7.13.12
>   ➤ YN0000: │ Published @babel/helper-define-map 7.13.12
>   ➤ YN0000: │ Published @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining 7.13.12
>   ➤ YN0000: │ Published @babel/preset-env 7.13.12
>   ➤ YN0000: │ Published @babel/helper-replace-supers 7.13.12
>   ➤ YN0000: │ Published @babel/plugin-transform-react-jsx 7.13.12
>   ➤ YN0000: │ Published @babel/helper-module-transforms 7.13.12
> ```
> 
> Source: https://github.com/babel/babel/runs/2167476229?check_suite_focus=true (expand the "Publish to npm" step).

As for the ordering, I was going by what npmjs claimed about the publish time.  Could be, as you suggest, an npmjs repo problem.
